packages/page-council/src/Overview/Vote.tsx

```diff
@@ -109,6 +109,20 @@ function Vote ({ electionsInfo }: Props): React.ReactElement<Props> {
             </Modal.Columns>
           </Modal.Content>
           <Modal.Actions onCancel={toggleVisible}>
+            {defaultVotes.length > 0 && (
+              <TxButton
+                accountId={accountId}
+                icon='trash-alt'
+                label={t<string>('Clear all')}
```

Comment:
I would actually suggest making this `isDisabled` instead of hidden.

So basically when you vote the first time you see it - it won't help, but at least some will remember and go there to clear.

Reply:
agreed, I'll do that.